### PR TITLE
Add WebSite X5 detection.

### PR DESCRIPTION
**Pull request by AlexByte**
Sites to test: 

- http://lansystems.it/
- https://www.tone2.com/
- http://www.deltapompen.com/

https://trends.builtwith.com/websitelist/Website-x5

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -9022,6 +9022,16 @@
 			"url": "/websale7/",
 			"website": "http://websale.de"
 		},
+		"WebSite X5": {
+			"cats": [
+				"20"
+			],
+			"meta": {
+				"generator": "Incomedia WebSite X5 (\\w+ [\\d.]+)\\;version:\\1"
```
Comment (jvoisin) [inline src/apps.json]: What about `"^Incomedia WebSite X5 (\\w+ [\\d.]+)\\;version:\\1"` to speed a bit the matching?

File: src/apps.json [hunk 2]
```diff
@@ -9022,6 +9022,16 @@
...
+			},
+			"icon": "WebSite X5.png",
+			"website": "http://websitex5.com"
```
Comment (jvoisin) [inline src/apps.json]: Please use `https` if possible ♥

Comment (AlexByte) [reply src/apps.json]: what does it affect?

Comment (jvoisin) [reply src/apps.json]: Uusing `https` instead of `http` is a good security practise :)

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**